src/classwork/04_assign/main.cpp

```diff
@@ -10,6 +15,19 @@ factorial.  Also, loop continues as long as user wants to.
 */
 int main() 
 {
+	
+	int num = 1;	
+
+	while (num != 0)
+	{
+		cout<<"Enter a Number to find its factorial: ";
+		cin>>num;    
+	
+		cout<<factorial(int num);
+	
+	}
 
```

Comment:
Should be:

auto choice = 'y';
	int num = 1;	

	while (choice == 'y')
	{
		cout<<"Enter a Number to find its factorial: ";
		cin>>num;   

		while(num < 1 || num > 10) 
		{
			cout<<"Number out of range, enter num from 1 to 10: ";
			cin>>num;
		}
	
		cout<<factorial(num)<<"\n";

		cout<<"Continue type y";
		cin>>choice;
	
	}